### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by nikich340** [VirtualDisplay/Virtual-Display-Driver#204]
### Which OS?

Win10 Pro

### Which release?

22H2

### Describe the bug

Can't select any other mode (in "List other modes") other than the one used by my 1st (real screen) - 1920x1080.
![image](https://github.com/user-attachments/assets/2208856e-7efa-4321-9117-6dd14a2b7910)

### Contact Details

nikich340@gmail.com

**Comment by itsmikethetech** [comment]
It seems like your user account may not have appropriate administrative privilages. Are you able to edit other monitors this way? I tried it myself and wasn't able to recreate the issue, but I'll try on another machine to be sure.

Can you try changing it in the Advanced Display Settings instead, as shown below?

![image](https://github.com/user-attachments/assets/e64d4f7c-913a-41e6-b07c-68eb6e592812)

**Comment by itsmikethetech** [comment]
Will test in Win 10 Pro soon. Needs a reinstall.

**Comment by nikich340** [comment]
Yes, I can change sample rate for physical and virtual display, but no way to change it's resolution.
![image](https://github.com/user-attachments/assets/1d6c489c-efa0-40ee-a1d2-1b0afc3652ba)

**Comment by nikich340** [comment]
![image](https://github.com/user-attachments/assets/c8f2347d-1ddb-4929-9155-009580fc4a97)

**Comment by bud3699** [comment]
Resolutions are in regular display settings, not advanced

**Comment by nikich340** [comment]
I have only default resolutions for physical display.
![image](https://github.com/user-attachments/assets/dd99611a-16c7-4e71-ab2d-c975f28bbd39)

**Comment by bud3699** [comment]
Yes at the top click the second monitor then go to change resolutions

**Comment by nikich340** [comment]
I am not sure how can I click on 2nd monitor as I am using duplicate variant so all their resolutions must be merged.
![image](https://github.com/user-attachments/assets/c3bc1d19-c46a-4244-b04c-2b48fa93f396)

**Comment by bud3699** [comment]
Well that will be why, you can't use a resolution on one if it's not on the other

**Comment by nikich340** [comment]
Wdym? If I connect my 4k TV via HDMI and select "duplicate" mode I actually can use 4k resolution. 
Sure, my 1st screen will downscale this to 1080p, but the point is to run applications in higher resolution than my main screen (to record video in higher resolution, for example).
What I expected from this driver is to emulate external 8k display like TV so I could use it for apps.
